src/libgraphviz/lib.rs

```diff
@@ -479,6 +494,16 @@ pub trait Labeller<'a,N,E> {
     }
 }
 
+/// Escape tags in such a way that it is suitable for inclusion in a
+/// Graphviz HTML label.
+pub fn escape_html(s: &str) -> String {
```

Comment:
... this isn't a great escape function, but I guess it's not really untrusted input.